experiments/ClimaEarth/user_io/postprocessing.jl

```diff
@@ -0,0 +1,123 @@
+## helpers for user-specified IO
+include("debug_plots.jl")
+include("diagnostics_plots.jl")
+
+"""
+    postprocess_sim(mode_type::AbstractSlabPlanetModeType, cs, postprocessing_vars)
+
+If they exist, perform conservation checks, for any slabplanet simulation, and then call
+`common_postprocessing` to perform common postprocessing tasks that are common to all simulation types.
+"""
+function postprocess_sim(mode_type::AbstractSlabPlanetModeType, cs, postprocessing_vars)
+    (;
+        dir_paths,
+        plot_diagnostics,
+        use_coupler_diagnostics,
+        output_default_diagnostics,
+        t_end,
+        conservation_softfail,
+        atmos_output_dir,
+        amip_diags_handler,
+        comms_ctx,
+    ) = postprocessing_vars
+
+    if !isnothing(cs.conservation_checks)
+        @info "Conservation Check Plots"
+        plot_global_conservation(
+            cs.conservation_checks.energy,
+            cs,
+            conservation_softfail,
+            figname1 = joinpath(dir_paths.artifacts, "total_energy_bucket.png"),
+            figname2 = joinpath(dir_paths.artifacts, "total_energy_log_bucket.png"),
+        )
+        plot_global_conservation(
+            cs.conservation_checks.water,
+            cs,
+            conservation_softfail,
+            figname1 = joinpath(dir_paths.artifacts, "total_water_bucket.png"),
+            figname2 = joinpath(dir_paths.artifacts, "total_water_log_bucket.png"),
+        )
+    end
+    common_postprocessing(cs, postprocessing_vars)
+end
+
+"""
+    postprocess_sim(mode_type::AMIP_mode, cs, postprocessing_vars)
+
+Conditionally plot AMIP diagnostics and call `common_postprocessing` to perform
+postprocessing tasks that are common to all simulation types.
+"""
+function postprocess_sim(mode_type::AMIP_mode, cs, postprocessing_vars)
+    (;
+        dir_paths,
+        plot_diagnostics,
+        use_coupler_diagnostics,
+        output_default_diagnostics,
+        t_end,
+        conservation_softfail,
```

Comment:
```suggestion
```

We don't need to extract args that won't be used in this function

Reply:
Same for the slabplanet method - e.g. we don't need the `amip_diags_handler` for that case